connect/runtime/src/test/java/org/apache/kafka/connect/runtime/AbstractHerderTest.java

```diff
@@ -1040,6 +1048,40 @@ public void testGetConnectorTypeWithEmptyConfig() {
         assertEquals(ConnectorType.UNKNOWN, herder.connectorType(Collections.emptyMap()));
     }
 
+    @Test
+    public void testGetConnectorOffsetsConnectorNotFound() {
```

Comment:
This test is failing now (probably because of the shift to use a snapshot in `AbstractHerder::connectorOffsets`?